rclpy/rclpy/client.py

```diff
@@ -44,12 +44,11 @@ def run(self):
         if sigint_gc_handle in guard_condition_ready_list:
             rclpy.utilities.shutdown()
             return
-        response = _rclpy.rclpy_take_response(
+        seq_and_response = _rclpy.rclpy_take_response(
```

Comment:
Could we use something lilke `[seq, response] = _rclpy.rclpy_take_response(...` for readability?

Reply:
Sure. It just means making rclpy_take_response return `(None, None)` on when there is nothing to take instead of None.

Reply:
Changed in 76a3d07d60585984df751186fd16d2c20e5d89c1

CI is running.